test/ApplicationSpecification.scala

```diff
@@ -21,7 +21,7 @@ trait ApplicationSpecification extends mutable.Specification with ResourceFinder
       )
 
     Play.start(app)
-    evolutionFor("collins")
+    //evolutionFor("collins")
```

Comment:
Is this accidentally commented out? I would expect we would want to run the evolutions for the specs.